.github/workflows/lint-check.yml

```
@@ -20,25 +20,7 @@ jobs:
           >&2 echo "Checking file $file..."
           import_lines=$(awk '/import/' "$file" | grep -v '# lint-check: ignore') || true
 
-          bad_lines=$(echo "$import_lines" | awk '!/https:/')
-          if [ -n "$bad_lines" ]; then
-            >&2 echo "Must import files from a URL!"
-            >&2 echo "The following lines are bad:"
-            >&2 echo "$bad_lines"
-            >&2 echo ""
-            EXITCODE=1
-          fi
-
-          bad_lines=$(echo "$import_lines" | awk '/https:\/\/raw.githubusercontent.com\/stjudecloud\/workflows/ && !/master/') || true
-          if [ -n "$bad_lines" ]; then
-            >&2 echo "Imports from this repo must use the master branch!"
-            >&2 echo "The following lines are bad:"
-            >&2 echo "$bad_lines"
-            >&2 echo ""
-            EXITCODE=1
-          fi
-          
-          bad_lines=$(echo "$import_lines" | awk '!/https:\/\/raw.githubusercontent.com\/stjudecloud\/workflows/ && /master/') || true
+          bad_lines=$(echo "$import_lines" | awk '!/https:\/\/raw.githubusercontent.com\/stjudecloud\/workflows/ && (/main/ || /master/)') || true
```

```suggestion
          bad_lines=$(echo "$import_lines" | awk '/http/ && (/main/ || /master/)') || true
```
Didn't actually test this awk pattern, but it looks like what we want. The current first half of the pattern we expect to be a no-op if we switch to relative imports.

This has me thinking..... if we switch to relative imports, than we should have a check that all `http` imports are to an external repo (i.e. not this one). I think the current check, plus the check I just suggested, should cover every case we want to exclude?

---

OK. I added a check to ensure nothing uses this repository's URL. I then updated the subsequent check to just look for `http` + `master` or `main` as you suggested above.